## Importação das bibliotecas

In [1]:
import pandas as pd

In [2]:
import numpy as np

In [4]:
from pandas_datareader import data

In [6]:
import matplotlib.pyplot as plt

In [7]:
import seaborn  as sns

In [8]:
import plotly.express as px

## Base de Dados com uma ação

In [15]:
import yfinance as yf

In [16]:
# gol_df = data.DataReader(name='GOLL4.SA', data_source='yahoo', start='2015-01-01')
gol_df = yf.download('GOLL4.SA', start='2015-01-01')

[*********************100%%**********************]  1 of 1 completed


In [17]:
gol_df

Open   High    Low  Close  Adj Close   Volume
Date                                                      
2015-01-02  14.97  15.20  14.61  14.99      14.99   901900
2015-01-05  14.90  15.01  14.41  14.85      14.85  1272500
2015-01-06  14.75  15.21  14.65  15.21      15.21  2183500
2015-01-07  15.15  15.33  14.55  14.55      14.55  1846900
2015-01-08  14.41  14.65  14.20  14.27      14.27  1569500
...           ...    ...    ...    ...        ...      ...
2024-05-06   1.37   1.37   1.33   1.33       1.33  4327200
2024-05-07   1.34   1.36   1.31   1.34       1.34  5336400
2024-05-08   1.34   1.34   1.30   1.30       1.30  3229800
2024-05-09   1.30   1.31   1.22   1.24       1.24  8012500
2024-05-10   1.24   1.27   1.22   1.27       1.27  6323500

[2326 rows x 6 columns]

analisando o primeiro dia da tabela (2015-01-02), temos que , na abertura da Bolsa, as ações da Gol estavam custando 14.97 , ao fechar 14.99. O preço mais alto foi 15.20, e o meor 14.61